### PR TITLE
added to javascript, added to html 

### PR DESCRIPTION
**Pull request by dvicj**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  @font-face {
+    font-family: poke;
+    src: url("./assets/fonts/pokemon-font.otf");
```
Comment (a-taghva) [inline assets/css/style.css]: @dvicj I think it should be: 
src: url("/assets/fonts/pokemon-font.otf")

File: assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    var pokeTwoType = pokemon.types[1].type.name; 
+    // if (pokeOneType === "grass") {
+    //     window.alert("Hi!");
+    // }
+    
+    currentPokemonEl.textContent = pokeOne;
+
+    currentTypeEl.textContent = pokeOneType + " / " + pokeTwoType; 
+
+    for(var i=0; i<3; i++) {
```
Comment (a-taghva) [inline assets/js/script.js]: @dvicj I didn't understand why you're using a loop here.

File: assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+//display Pokemon image
+var pokemonImage = function (pokemon) {
+    var pokeNumber = pokemon.id; 
+    console.log(pokemon.id);
+    var pokeImage = document.createElement("img");
+    pokeImage.setAttribute("style", "width:250px;height:250px;")
+
+    pokeImage.srcset = `https://pokeres.bastionbot.org/images/pokemon/` + pokeNumber + `.png`;
```
Comment (a-taghva) [inline assets/js/script.js]: @dvicj If you're using backticks, I think it's better to use it this way:
`https://pokeres.bastionbot.org/images/pokemon/${pokeNumber}.png`

File: assets/css/style.css (hunk 1 above)
Comment (malloryfaria) [reply assets/css/style.css]: I think it should be fine either way with or without the dot but let me know if I should update 💯

Comment (a-taghva) [reply assets/css/style.css]: ./ means current directory
/ means root directory

File: assets/js/script.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+//search for city 
+    // find weather condition - ie. sunny, cloudy, snowy etc.
+        //find pokemon types based on weather 
+            // if sunny -- find fire pokemon, display 10 
+
+let cardEl = $(".poke-card");
+let infoEl = $(".poke-info");
+let imageEl = $("#pokemon-image");
+      
+$(cardEl).on('mousemove', (e) => {
```
Comment (a-taghva) [inline assets/js/script.js]: it's better to indent lines 96-102, 106, 110-114

File: assets/css/style.css (hunk 1 above)
Comment (malloryfaria) [reply assets/css/style.css]: Ahhh okay good point! I'll update it.

Comment (a-taghva) [reply assets/css/style.css]: I'm a little confused, is this your file or Devin's?

**Review by malloryfaria — COMMENTED** [review]
The tailwind css link isn't on the html in this one but looks great otherwise!

File: assets/css/style.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  @font-face {
+    font-family: poke;
+    src: url("/Poke-Parte/assets/fonts/pokemon-font.otf");
```
Comment (a-taghva) [inline assets/css/style.css]: @dvicj The url should be:
src: url("/assets/fonts/pokemon-font.otf");

File: index.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (a-taghva) [inline index.html:1]: @dvicj About the html file if you push this to the main branch Mallory's file will be removed, but her file is complete and more structured and it also has the link to tailwind.css.
So I think it's better that you copy her html file and add the POKE CARDS section to it, then push your html file.

**Pull request closed without merging**